test/unittests/parser_test.cpp

```diff
@@ -624,11 +624,11 @@ TEST(parser, global_multi_const_inited)
     EXPECT_FALSE(module.globalsec[0].type.is_mutable);
     EXPECT_EQ(module.globalsec[0].type.value_type, ValType::i32);
     EXPECT_EQ(module.globalsec[0].expression.kind, ConstantExpression::Kind::Constant);
-    EXPECT_EQ(module.globalsec[0].expression.value.constant, 0x01);
+    EXPECT_EQ(as_uint32(module.globalsec[0].expression.value.constant), 1);
     EXPECT_TRUE(module.globalsec[1].type.is_mutable);
     EXPECT_EQ(module.globalsec[1].type.value_type, ValType::i32);
     EXPECT_EQ(module.globalsec[1].expression.kind, ConstantExpression::Kind::Constant);
-    EXPECT_EQ(module.globalsec[1].expression.value.constant, uint32_t(-1));
+    EXPECT_EQ(as_uint32(module.globalsec[1].expression.value.constant), -1);
```

Comment:
```suggestion
    EXPECT_EQ(as_uint32(module.globalsec[1].expression.value.constant), uint32_t(-1));
```
This should stay as otherwise implicit conversion is needed.

Reply:
Couldn't we just use `as_int32(..), -1` ?

Reply:
In theory you can, but I don't see strong point for it. I prefer to refer to `i32` as `uint32_t` unless some heavy signed arithmetic is involved. Here you can just replace it with `0xffffffff`.

Reply:
Alright, removed it.